client/rest-high-level/src/test/java/org/elasticsearch/client/TransformIT.java

```diff
@@ -82,18 +82,18 @@ private void createIndex(String indexName) throws IOException {
 
         XContentBuilder builder = jsonBuilder();
         builder.startObject()
-                .startObject("properties")
-                .startObject("timestamp")
-                .field("type", "date")
-                .endObject()
-                .startObject("user_id")
-                .field("type", "keyword")
-                .endObject()
-                .startObject("stars")
-                .field("type", "integer")
-                .endObject()
-                .endObject()
-                .endObject();
+            .startObject("properties")
```

Comment:
Is there any integration test added for the new functionality? I couldn't find it in this PR and I think it would be beneficial.